discord/cog.py

```diff
@@ -739,16 +742,41 @@ def _resolve_name(self, name: str, package: Optional[str]) -> str:
         except ImportError:
             raise errors.ExtensionNotFound(name)
 
-    def load_extension(self, name: str, *, package: Optional[str] = None) -> None:
-        """Loads an extension.
+    def load_extensions_from_path(
+        self, *paths: Union[str, pathlib.Path], recursive: bool = False, must_exist: bool = True
+    ) -> None:
+        if len(paths) > 1 or not paths:
+            for path_ in paths:
+                self.load_extensions_from_path(path_, recursive=recursive, must_exist=must_exist)
+            return
+
+        path = paths[0]
+
+        if isinstance(path, str):
+            path = pathlib.Path(path)
+
+        try:
+            path = path.resolve().relative_to(pathlib.Path.cwd())
+        except ValueError:
+            raise ValueError(f"'{path}' must be relative to the working directory") from None
```

Comment:
`pathlib.Path` should have a function that allows for you to check if the path provided is absolute. Use that instead of this check.